src/Http/Controllers/Character/MarketController.php

```diff
@@ -57,18 +58,37 @@ public function getMarket(int $character_id)
     public function getMarketData(int $character_id)
     {
 
-        $orders = $this->getCharacterMarketOrders($character_id, false);
+        if (! request()->has('all_linked_characters'))
+            return abort(500);
```

Comment:
shouldn't rather be a 400 ? @leonjza

Reply:
Qu1ckkk said here to use 500: https://eve-seat.slack.com/archives/C0H3VGH4H/p1543645324006200